lib/cognition/plugins/default.rb

```diff
@@ -15,11 +15,13 @@ def pong(msg, match_data = nil)
       end
 
       def help(msg, match_data = nil)
+        type = msg.metadata[:type] ||= "text"
         if match_data['command'].empty?
-          Cognition.help.join("\n")
+          @help = Cognition.help
         else
-          Cognition.help.find_all { |text| text.match match_data[:command] }.join("\n")
+          @help = Cognition.help.find_all { |text| text.match match_data[:command] }
         end
+        render(type: type)
```

Comment:
This uses the template system to render a text template by default, or an html template if the implementing app passes `:type` in with the metadata hash.